src/display.jl

```diff
@@ -1,6 +1,6 @@
 
-import Base.writemime
-export writemime
+import Base.show
+export show
```

Comment:
These two lines are not necessary. Instead I'd recommend to explicitly use the parent module always when extending methods, because it leads to clearer code. Instead write:
```julia
function Base.show(...
```

Reply:
Sounds good. I changed it in the newest commit